contracts/FullCheckpoint.sol

```diff
@@ -278,17 +270,25 @@ contract FullCheckpoint {
         }
     }
 
+    /* @dev Confirm all ancestor unconfirmed block
```

Comment:
can you explain to readers that the word "confirm"="commit"="finalize" ?

Reply:
sure